package.json

```diff
@@ -32,12 +32,15 @@
     "@playwright/test": "1.56.1",
     "@wordpress/e2e-test-utils-playwright": "^1.35.0",
     "@wordpress/env": "^10.35.0",
-    "@wordpress/scripts": "^30.17.0",
+    "@wordpress/scripts": "^31.1.0",
     "ajv": "^8.17.1"
   },
   "dependencies": {
     "npm": "^11.4.1"
   },
+  "overrides": {
```

Comment:
@ShawnTurple is there a problem with using this approach? It seems wordpress/scripts is using an unsafe version of webpack-dev-server. We could track this override as an issue, otherwise we must simply wait for wordpress/scripts to update their dependencies.